qtop_py/contrib/sger_dvv_out.ref

```diff
@@ -1,36 +1,34 @@
 
-
./qtop.py ## Queueing System report tool. For feedback and updates, see: https://github.com/qtop/qtop
-Please try it with watch: /home/sfranky/PycharmProjects/qtop/qtop.py -s <SOURCEDIR> -w [<every_nr_of_sec>]
-...and thank you for watching ;)
 
-[1;30m===> [0;m[1;37mJob accounting summary[0;m[1;30m <=== [0;m[1;37m2016-11-15 21:46:58[0;m WORKDIR = /home/sfranky/PycharmProjects/qtop
-[1;33mSummary[0;m: Total:[1;31m17[0;m Up:[1;31m17[0;m Free:[1;31m4[0;m [1;31mNodes[0;m | [1;32m62[0;m/[1;32m408[0;m [1;32mcores[0;m |   [1;34m61[0;m+[1;34m31[0;m [1;34mjobs[0;m (R + Q) 
- [1;33mQueues[0;m:  [1;31malice[0;m: [1;31m59[0;m | [0;35mwhole[0;m: [0;35m2[0;m | [1;35mcta[0;m: [1;35m0[0;m | ctaprod: 0 | drihm: 0 | dteam.cg: 0 | gisela.cg: 0 | ibchem.cg: 0 | ibearth.cg: 0 | ibeng.cg: 0 | iber.cg: 0 | ibhpc.cg: 0 | ibict.cg: 0 | iblife.cg: 0 | ibops.cg: 0 | ibphys.cg: 0 | ibsocial.cg: 0 | ibtut.cg: 0 | ops.cg: 0 | sqm.test: 0 | [1;33mPending[0;m: [1;33m0[0;m + [1;33m31[0;m | [0;31m* implies blocked[0;m
+[1;36m./qtop.py     ## Queueing System report tool. Press ? for help[0;m 
+v0.9.20161222 ## For feedback and updates, see: [1;36mhttps://github.com/qtop/qtop[0;m
```

Comment:
hm, this is hardwired in the test, which could be an issue in the future; 
something to think of, but not a blocker for now